modules/python/python-logmsg.c

```diff
@@ -100,7 +100,7 @@ _py_log_message_getattr(PyObject *o, PyObject *key)
 }
 
 static int
-_py_log_message_setattr(PyObject *o, PyObject *key, PyObject *value)
+_py_log_message_ass_subscript(PyObject *o, PyObject *key, PyObject *value)
```

Comment:
What does `ass_subscript` abbreviate?

Reply:
no clue, https://docs.python.org/3/c-api/typeobj.html#c.PyMappingMethods

Reply:
Oh ok, so it comes from the official cython api.